crates/transaction-emitter/src/lib.rs

```diff
@@ -783,7 +815,7 @@ async fn wait_for_accounts_sequence(
             }
         }
 
-        time::sleep(Duration::from_millis(500)).await;
```

Comment:
What's the reasoning behind changing this?

Reply:
Just faster turnaround time really.